src/main/java/core/basesyntax/figures/Rectangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.figures;
+
+public class Rectangle extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line above the class declaration.